proto/pdpb.proto

```diff
@@ -800,6 +800,7 @@ message ScatterRegionRequest {
     // If regions_id is defined, the region_id would be ignored.
     repeated uint64 regions_id = 6;
     uint64 retry_limit = 7;
+    bool skip_limit_check = 8;
```

Comment:
maybe `skip_store_limit`  or `skip_store_limit_restriction`

`limit_check` may confuse with `retry_limit`

Reply:
I prefer a more general name but if we don't find one, `skip_store_limit` is ok to me.